src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -58,6 +58,20 @@ static void LogFindResults(ActiveLicenseFindResult result)
 
                 Logger.Info(report.ToString());
             }
+
+#if REGISTRYLICENSESOURCE
+            if (result.Location.StartsWith("HKEY_"))
```

Comment:
do we ensure that the file based location always wins over the registry key if a license on both locations has been found?